R/get_datagrid.R

```diff
@@ -1,9 +1,13 @@
 #' Create a reference grid
 #'
 #' Create a reference matrix, useful for visualisation, with evenly spread and
-#' combined values. Usually used to make generate predictions using [get_predicted()].
-#' See this [vignette](https://easystats.github.io/modelbased/articles/visualisation_matrix.html)
+#' combined values. Usually used to make generate predictions using
+#' [get_predicted()]. See this
+#' [vignette](https://easystats.github.io/modelbased/articles/visualisation_matrix.html)
 #' for a tutorial on how to create a visualisation matrix using this function.
+#' \cr\cr
+#' Alternatively, these can also be used to extract the "grid" columns from
+#' objects generated by `emmeans` and `marginaleffects`.
```

Comment:
Standard package formatting (with `\pkg`) is usually bold, I think this is (mostly) consistent throughout the docs, so please use `*` or `**` for packages.